lib/merge.js

```diff
@@ -1,4 +1,5 @@
 const { logger, retry } = require("./common");
+var _ = require('lodash');
```

Comment:
Thanks for the PR, it doesn't look bad at all! :) One note, we currently only depend on lodash for devDependencies (e.g. eslint needs it). If possible, a solution without lodash would be nice, but otherwise we would need to add lodash to the regular dependencies (yarn add lodash)

Reply:
I indeed realized that lodash was quite a big dependency for this simple use case. I now use [object-resolve-path](https://github.com/capaj/object-resolve-path).